sdk/samples/sender_app.c

```diff
@@ -185,20 +106,28 @@ int main(int argc, char** argv)
         { "send_ip", required_argument, NULL, 's' },
         { "send_port", required_argument, NULL, 'p' },
         { "protocol", required_argument, NULL, 'o' },
-        { "number", required_argument, NULL, 'n' },
-        { "file", required_argument, NULL, 'b' },
         { "type", required_argument, NULL, 't' },
         { "socketpath", required_argument, NULL, 'k' },
         { "master", required_argument, NULL, 'm' },
         { "interfaceid", required_argument, NULL, 'd' },
-        { "loop", required_argument, NULL, 'l' },
+        { "file", required_argument, NULL, 'b' },
         { "pix_fmt", required_argument, NULL, 'x' },
+        { "audio_type", required_argument, NULL, 'a' },
+        { "audio_format", required_argument, NULL, 'j' },
+        { "audio_sampling", required_argument, NULL, 'g' },
+        { "audio_ptime", required_argument, NULL, 'e' },
+        { "audio_channels", required_argument, NULL, 'c' },
+        { "anc_type", required_argument, NULL, 'q' },
+        { "number", required_argument, NULL, 'n' },
+        { "loop", required_argument, NULL, 'l' },
```

Comment:
Adding arguments here? Please reflect the change in README.md for the user to be familiar with them.

Reply:
Added to README.md (with accordance to usage/help)